docs/recipes/event-delegation.md

```diff
@@ -0,0 +1,45 @@
+# Event delegation 
+
+In the 1st example we use `most.fromEvent` to create a stream from
+a `click` event. 
+
+By adding `filter` in conjunction with [element matches](https://developer.mozilla.org/en-US/docs/Web/API/Element/matches)
+we can specify which element we want to observe and in this case get the `textContent`
+from the element.
+
+The 2nd example is similar, however we use jQuery to handle event delegation.
+
+We use `most.create` to produce a push-stream off of those click events.
+Essentially each time a user clicks on the `.inner` that event is added to the
+stream which can be observed and processed in a similar manner to the 1st example.
+
+It should be noted that the `dispose` function which calls `container.off` only happens
+if the implementor would call `end()`, otherwise the stream will not end and
```

Comment:
Technically, the `dispose` will be called when there's no *demand* for events.  That can happen in several ways.  One way is when the stream producer (i.e. the function passed to `most.create`) explicitly calls `end`.  Another is when all consumers have "lost interest".  For example, if 2 callers start consuming the stream, and the first uses `.take(5)` and the second uses `.take(10)`, the first 5 events will be delivered to both consumers.  Then, the first loses interest (because of `take(5)`).  Since the second consumer is still interested, `dispose` is not called yet.  The second consumer receives 5 more events, then loses interest (because of `take(10)`).  At that point, `dispose` is called because there is no longer any demand for events.

That's probably waaaayyyyy to long-winded to explain in the recipe :)  So, one option would be to link to the [docs for `most.create`](https://github.com/cujojs/most/blob/master/docs/api.md#mostcreate) which explain how it works.